### PR TITLE
ondaylongpress

### PR DESCRIPTION
**Pull request by tuur155**
I made ondaylongpress work

### CODE REVIEW

File: example/android/.project [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Inbal-Tish) [inline example/android/.project]: This file should not be changed

File: example/android/.settings/org.eclipse.buildship.core.prefs [hunk 2]
```diff
@@ -0,0 +1,2 @@
+connection.project.dir=
```
Comment (Inbal-Tish) [inline example/android/.settings/org.eclipse.buildship.core.prefs]: This file should not be changed

File: src/calendar-list/index.js [hunk 3]
```diff
@@ -1,17 +1,17 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react';
```
Comment (Inbal-Tish) [inline src/calendar-list/index.js]: Please remove all formatting changes

File: src/calendar-list/index.js [hunk 4]
```diff
@@ -47,7 +46,7 @@ class CalendarList extends Component {
   static defaultProps = {
     horizontal: false,
     calendarWidth: width,
-    calendarHeight: 360,
+    calendarHeight: 390,
```
Comment (Inbal-Tish) [inline src/calendar-list/index.js]: Why are you changing the default value?

File: src/calendar-list/item.js [hunk 5]
```diff
@@ -44,11 +44,13 @@ class CalendarListItem extends Component {
           dayComponent={this.props.dayComponent}
           disabledByDefault={this.props.disabledByDefault}
           showWeekNumbers={this.props.showWeekNumbers}
+          uren={this.props.uren}
+          bezettingColor={this.props.bezettingColor}
```
Comment (Inbal-Tish) [inline src/calendar-list/item.js]: What are these two props? What language is that?

File: src/calendar/day/multi-period/index.js [hunk 6]
```diff
@@ -14,21 +15,27 @@ class Day extends Component {
     theme: PropTypes.object,
     marking: PropTypes.any,
     onPress: PropTypes.func,
+    onLongPress: PropTypes.func,
     date: PropTypes.object,
+    uren: PropTypes.string,
```
Comment (Inbal-Tish) [inline src/calendar/day/multi-period/index.js]: ???

**Comment by Inbal-Tish** [comment]
@tuur155 Hi. This PR is unreadable. Also, onLongPress is already added to day components. Please open a new PR if you have another issue to solve. Thanks

**Comment by tuur155** [comment]
Hi, a lot of personal commits came unintended with this PR. I'm not so good with Github yet. I wanted to do this PR a few months ago to fix onDayOnLongPress on multi period days. I guess i accidentally pushed it again or something. My bad.

**Comment by Inbal-Tish** [comment]
@tuur155 Please checkout version 1.240.0 where I added the 'onLongPress' to Day multi-period. Feel free to open a new one if you need it. Thanks!

**Pull request closed without merging**